api/jobs.go

```diff
@@ -97,6 +92,9 @@ func (api *API) CreateJobHandler(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	newJob.Links.Self = host + "/v1" + newJob.Links.Self
```

Comment:
**COMMENT 2**
You should be not doing this manipulation of data in the handler layer. To achieve this, you can add the `SEARCH_REINDEX_API_ADDR` in the `config` and then use that within the CreateJob or whatever - instead of `host`. Something similar is done in the `dp-dataset-api`